### PR TITLE
Change address.codesize type to uint256

### PR DESCRIPTION
**Pull request by iamdefinitelyahuman**
### What I did
Change the type of `address.codesize` to `uint256`.

### How I did it
With style and grace.

### How to verify it
Run the tests.

### Cute Animal Picture
image

### CODE REVIEW

File: vyper/parser/expr.py [hunk 1]
```diff
@@ -328,8 +328,11 @@ def parse_Attribute(self):
             addr = Expr.parse_value_expr(self.expr.value, self.context)
             if is_base_type(addr.typ, "address"):
                 if self.expr.attr == "codesize":
-                    eval_code = ["extcodesize", addr]
-                    output_type = "int128"
+                    if self.expr.value.id == "self":
+                        eval_code = ["codesize"]
```
Comment (fubuloubu) [inline vyper/parser/expr.py:332]: This was added in Istanbul, right?

Comment (fubuloubu) [reply vyper/parser/expr.py:332]: Yes, `EXTCODEHASH` was added in Istanbul: https://eips.ethereum.org/EIPS/eip-1884

Comment (fubuloubu) [reply vyper/parser/expr.py:332]: This should be `... or evm_version < Istanbul`

And then add test cases for this behavior before/after Istanbul.

I think this feature was added recently, so maybe we missed this in the VIP?

Comment (iamdefinitelyahuman) [reply vyper/parser/expr.py:332]: No.  You might be thinking of `EXTCODEHASH` that was added in constantinople?

Comment (fubuloubu) [reply vyper/parser/expr.py:332]: Ah, yes totally right. Disregard then.

**Pull request merged**